test/test.js

```diff
@@ -0,0 +1,49 @@
+﻿const mocha = require('mocha');
+const chai = require('chai');
+const program = require('../words');
+const assert = chai.assert;
+chai.config.truncateThreshold = 0;
+describe('Test words function', () => {
+    it('should handle specified input', () => {
+        let testcase = program.words('olly olly in come free');
+        assert.deepEqual(testcase, { olly: 2, in: 1, come: 1, free: 1 });
+    });
+    it('should handle more complex input', () => {
```

Comment:
This test case is not exactly different from the first test case. It isn't more complex. It only has more words. If you wanted to specify a more complex sentence, perhaps, you'd have configured your function to handle `special characters` or `numbers` sandwiched in between valid words in the sentence, then you either make the function ignore them or recognize them as well. Reply on this thread if you need more clarification about this.

Reply:
Well understood. Thanks for the hint. I thought some words might complicate issues like in the sentence: "Friend's house friend", hence I decided to ignore special characters. Would therefore prefer to remove this test case since it seems redundant.